frontend/src/hooks/useKeycloakWrapper.tsx

```diff
@@ -49,7 +49,7 @@ export interface IKeycloak {
 /**
  * Provides extension methods to interact with the `keycloak` object.
  */
-function useKeycloakWrapper(): IKeycloak {
+export function useKeycloakWrapper(): IKeycloak {
```

Comment:
The general recommendation these days is to not use default exports, but I'm not in the know on whether it truly matters.  Either way, we can do both.

Reply:
I'll look into this, I don't really like any recommendation if I don't understand the rationale behind it.

Reply:
Adding it this way doesn't change the rest of our code, it just lets us use the `import { name } from name` and `import name from name`